moveit_ros/warehouse/warehouse/src/import_from_text.cpp

```diff
@@ -256,9 +358,12 @@ int main(int argc, char** argv)
   {
     std::ifstream fin(vm["queries"].as<std::string>().c_str());
     if (fin.good() && !fin.eof())
-      parseQueries(fin, &psm, &rs, &cs);
+    {
+      parseQueries(fin, &psm, &rs, &cs, &pss);
+    }
     fin.close();
   }
 
+  ros::waitForShutdown();
```

Comment:
I believe the node should exit after it finished.
Why do you want to block the node until someone shuts it down via Ctrl-C?